CNV- 25428: RN dynamic SSH key (#64576)
openshift/openshift-docs: virt/release_notes/virt-4-14-release-notes.adoc

* As a cluster administrator, you can back up and restore applications running on {VirtProductName} by using the xref:../../virt/backup_restore/virt-backup-restore-overview.adoc#virt-backup-restore-overview[OpenShift API for Data Protection (OADP)].

//CNV-29935

> * You can add a static xref:../../virt/virtual_machines/virt-accessing-vm-ssh#static-key-management-vm[authorized SSH key] to a project by using the web console. The key is subsequently added to all VMs that you create in the project.

Reviewer: Tiny nit, but up to you
```suggestion
* You can add a static xref:../../virt/virtual_machines/virt-accessing-vm-ssh#static-key-management-vm[authorized SSH key] to a project using the web console. The key is then added to all VMs that you create in the project.
```
